instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/Java17RuntimeMetrics.java

```diff
@@ -21,17 +21,17 @@
 import jdk.jfr.consumer.RecordingStream;
 
 /** The entry point class for runtime metrics support using JFR and JMX. */
-public final class RuntimeMetrics implements Closeable {
+public final class Java17RuntimeMetrics implements Closeable {
```

Comment:
is there a particular reason for this rename? I think `RuntimeMetrics` and `RuntimeMetricsBuilder` (or maybe better `RuntimeTelemetry` and `RuntimeTelemetryBuilder`) seem more consistent with other library telemetry builders

Reply:
It was suggested by @jeanbisutti to disambiguate between 2 classes with the same name - that are both used in the spring config

Reply:
reverted the name changes @trask